auto_test:
  ros__parameters:
    test_cases:
      - MakeGoalStraight
    repeats: 1
    loop: true
